man/fido_cred_new.3

```diff
@@ -265,6 +274,20 @@ The corresponding length can be obtained by
 and
 .Fn fido_cred_attstmt_len .
 .Pp
+The
+.Fn fido_cred_x5c_list_count
+function returns the length of the x509 certificate chain in
+.Fa cred
+and the
+.Fn fido_cred_x5c_list_ptr ,
+.Fn fido_cred_x5c_list_len ,
+functions return a pointer to and length of the x509 certificate at index
+.Fa idx
+respectively.
+Please note that the leaf certificate has an
+.Fa idx
+(index) value of 0.
+.Pp
```

Comment:
LGTM. For additional clarity, I'd suggest mentioning that:

- fido_cred_x5c_{ptr,len}(c) == fido_cred_x5c_list_{ptr,len}(c, 0);
- fido_cred_x5c_list_{ptr,len}(c, i) where i >= fido_cred_x5c_list_count(c) returns NULL/0.

Reply:
Pushed commit d1da1e3 that should cover this.